CSETWebNg/src/assets/config.json

```diff
@@ -1,5 +1,6 @@
 {
     "isCsetOnline": false,
+    "isCyberFlorida": false,
```

Comment:
Do we even need this in the config file if its getting set in the config service?